packages/acceptit-GUI.package/ACBrowser.class/instance/findCategories.st

```diff
@@ -0,0 +1,11 @@
+accessing-ui
+findCategories
+	| visible |
+	visible := Set new.
+	self baseClass withAllSubclassesDo: [ :each |
+		each category ifNotNil: [ :category |
+			visible add: category ] ].
+	^ Array streamContents: [ :stream |
+		self environment organization categories do: [ :each |
+			(visible includes: each)
+				ifTrue: [ stream nextPut: each ] ] ].
```

Comment:
why not just:
```
visible do [:each | stream nextPut: each]
```
instead of the includes etc.?
Also a small unit test that creates a ACUserStory subclass with a category and checks if this List includes that category would be nice.